vault/values.yaml

```diff
@@ -0,0 +1,31 @@
+vault:
+  injector:
+    enabled: false
+  server:
+    serviceAccount:
+      createSecret: true
+    ha:
+      enabled: false
+      raft:
+        enabled: false
```

Comment:
It is generally not recommended to use nesting that much in helm unless there's a valid reason for it. I recommend 
1) flattening the structure to something like `vaultInjector: true` instead of vault.injector.enabled
2) hide default settings from values.yaml and instead document them in README

Reply:
These values are belong to dependended 'vault' chart, so i don't know if we can't change them 🤔